core/src/banking_stage/consume_worker.rs

```diff
@@ -416,6 +447,16 @@ impl ConsumeWorkerCountMetrics {
                     .swap(0, Ordering::Relaxed),
                 i64
             ),
+            (
+                "min_prioritization_fees",
+                self.min_prioritization_fees.swap(0, Ordering::Relaxed),
```

Comment:
This isn't being reset correctly, and we'd always see 0 as the min after first iteration. Need to reset to u64::MAX (the original value):
```suggestion
                self.min_prioritization_fees.swap(u64::MAX, Ordering::Relaxed),
```